Makefile

```diff
@@ -11,8 +11,8 @@ endif
 
 GRPC_SERVICES_DIR=src/grpc/autogen
 
-GATEWAY_RS_VSN ?= "v1.0.0-alpha.31"
-GWMP_MUX_VSN ?= "v0.9.4"
+GATEWAY_RS_VSN ?= "8bf086618787b7eacf64abefdadf65a383e4c954"
```

Comment:
We're switching to commit hash instead of tag?

Reply:
we've been using commit hash whenever there was desire to avoid tagging; @madninja doesn't want to generate a new tag and kick off a redeploy/update of all light gateways just for a miner release